controller/volume_controller_test.go

```diff
@@ -913,7 +925,7 @@ func (s *TestSuite) TestVolumeLifeCycle(c *C) {
 	}
 	tc.expectReplicas = expectRs
 	testCases["volume attaching - start replicas - manager down"] = tc
-	//s.runTestCases(c, testCases)
+	s.runTestCases(c, testCases)
```

Comment:
It looks like we are running most volume controller test cases twice due to this line duplicating 1058. Confirmed by running `go test -v ./controller/...` on this branch and searching for "volume attaching - start replicas - manager down", which runs twice and "replica rebuilding - delay replica replenishment", which runs once. I immediately see a reason for the duplication.

Reply:
Nice catch! Thank you. Fixed